qemu/tests/cfg/nvdimm.cfg

```diff
@@ -26,7 +26,7 @@
     post_command = "rm -rf ${nv_backend}"
     mem_devs = mem1
     size_mem = 1G
-    share_mem = on
+    share_mem = yes
```

Comment:
To make it 'True/False' for QEMU 5.0 ?  Have you try it with RHEL7 ?

Reply:
Yes, it works well on RHEL7 as well.